youtube_dl/extractor/ceskatelevize.py

```diff
@@ -12,70 +12,243 @@
     ExtractorError,
     float_or_none,
     sanitized_Request,
-    unescapeHTML,
-    update_url_query,
+    str_or_none,
     urlencode_postdata,
     USER_AGENTS,
 )
 
+try:
+    from ..utils import traverse_obj
+except ImportError:
+    from ..compat import (compat_str, compat_collections_abc)
+    from ..utils import int_or_none
+
+    def traverse_obj(obj, *path_list, **kw):
```

Comment:
if u r going to port `traverse_obj`, port it properly to utils imho. It is very useful and superseeds most instances of `try_get` and `dict_get` while being able to do much more.

If for just this extractor, you can easily replace traverse_obj with equivalent `try_get`s

Reply:
Thanks. I am going to port `traverse_obj()`, so as to make it easy to port extractors back and forth. This is actually my best current effort at a candidate for utils, but I had in mind at some point in the future to pull various things from yt-dlp/utils in a separate PR (that will delete this version). So this is a stopgap to get this extractor ported without changing the extractor code unnecessarily. Although extractors might not need all the features, I'd rather not have to change tests or disappoint someone who wanted to upgrade the simple `--print` proposed elsewhere.